TestGeometricSeqPhenotype.java

```diff
@@ -0,0 +1,199 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Map;
+import java.util.Random;
+import java.util.Scanner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * A class for testing the methods in GeometricSeqPhenotype.java
+ *
+ * @author Thien Tran
+ */
+public class TestGeometricSeqPhenotype {
+    // Private fields
+    private GeometricSeqPhenotype emptyPheno; // Default constructor
+    private GeometricSeqPhenotype simplePheno; // Give a specific sequence
+    private GeometricSeqPhenotype simplePheno2; // Give a specific sequence
+
+    GeometricSeqPhenotype [] history;
+
+    /**
+     * Define variables needed for multiple tests.
+     */
+    @Before
+    public void setUp() {
+        // initialize static parameters
+        Parameters.load();
+        Parameters.initialize();
+
+        Random r = new Random();
+        long seed = 5;
+        r.setSeed(seed);
+        emptyPheno = new GeometricSeqPhenotype();
+        simplePheno = new GeometricSeqPhenotype(0.0, 0.0, new char[]{'T', 'G', 'C', 'A', 'T', 'C'});
+
+        simplePheno2 = new GeometricSeqPhenotype(20.0, 10.0, new char[]{'T', 'G', 'C', 'A', 'T', 'C'});
+
+        // Create a small history of GeometricSeq phenotypes
+        history = new GeometricSeqPhenotype[3];
+
+        history[0] = new GeometricSeqPhenotype(1.0, 2.0, new char[]{'T', 'G', 'C', 'A', 'T', 'C'});
```

Comment:
Why do these viruses have different locations in antigenic space, but the same sequence? Is it part of the test? Or does it just not matter?

Reply:
`history[]` is used to test `riskOfInfection()` and `distance()`. These two methods only need the locations of the viruses in antigenic space, so the sequences don't matter. However, I can change it to make things clearer.